### PR TITLE
[WIP] NFV: Process performance test results

### PR DESCRIPTION
**Pull request by asmorodskyi**

### CODE REVIEW

**Comment by asmorodskyi** [comment]
@jlausuch it is only first part , with upload logs , but you can give your early feedback already

**Comment by jlausuch** [comment]
looks good for the upload_logs part

**Comment by jlausuch** [comment]
you can add
    pip install requests
somewhere in prepare_env.pm

File: tests/nfv/process_perf_results.pm [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+    record_info("Package and upload logs");
+    my $logs_path = '/tmp/vsperf_logs.tar.gz';
+    my $results_folder = '/tmp';
+    prepare_testdata_temp();
+    assert_script_run("tar -czvf $logs_path $results_folder/results_*");
+    upload_logs($logs_path, failok => 1);
+    assert_script_run('wget https://raw.githubusercontent.com/asmorodskyi/vsperf_influxdb_connector/parse_folders/export.py');
+    assert_script_run('chmod +x export.py');
+    assert_script_run(sprintf('./export.py --parsefolder "%s" --targeturl http://10.86.0.128:8086 --os_version %s --os_build %s --vswitch_version %s --openqa_url %s',$results_folder,get_var('VERSION'),get_var('BUILD'),'1','1'));
```
Comment (asmorodskyi) [inline tests/nfv/process_perf_results.pm]: TODO: generate openqa job url ( or get it from some variable ?  and add vswitch_version

File: tests/nfv/process_perf_results.pm [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+#       - package and upload  perf logs to openQA job
+#       - fill up results DB with data 
+# Maintainer: Anton Smorodskyi <asmorodskyi@suse.com>, Jose Lausuch <jalausuch@suse.com>
+
+use base "opensusebasetest";
+use testapi;
+use strict;
+use lockapi;
+
+sub prepare_testdata_temp {
```
Comment (asmorodskyi) [inline tests/nfv/process_perf_results.pm]: TODO : function to fake results in vm runs , need to delete before merge

File: tests/nfv/process_perf_results.pm (hunk 1 above)
Comment (jlausuch) [reply tests/nfv/process_perf_results.pm]: maybe we can leave it for a later PR. But I've seen that the env variable NAME contains the test ID.
Examples:
1) https://openqa.suse.de/tests/1898583#settings
    NAME = 01898583-sle-12-SP4-Server-DVD-x86_64-Build0325-nfv_master_virtual@64bit

2) http://loewe.arch.suse.de/tests/1098#settings
    NAME = 00001098-sle-12-SP4-Server-DVD-x86_64-Build0324-nfv_master@64bit-ipmi

We just need to remove the 0's at the beginning and append it to "https://openqa.suse.de/tests/
Use this code:

    test_name = get_required_var("NAME");
    my ($test_id) = $test_name =~ m{^([^-]*)};
    $test_id =~ s/^0+//;
    my $test_url = "http://openqa.suse.de/tests/$test_id";
    print "$test_url\n";



OVS version can be taken from the command ovs-vsctl --version
Example of output:

$ ovs-vswitchd --version
ovs-vswitchd (Open vSwitch) 2.9.0

Just grab the first line of the output and take the last word separated by spaces: 
$ ovs-vswitchd --version|head -1|awk '{print $NF}'
2.9.0

**Pull request closed without merging**